backend/ctgov/api/views.py

```diff
@@ -622,22 +621,16 @@ def filter_study_roa(request):
 
     q_study_roa = Q()
     if study_roa:
-        roa_list = str(study_roa).split(",")
```

Comment:
This and the string replacements are not necessary as `study_roa` is already a python list.